example/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
 
     <uses-sdk
-        android:minSdkVersion="16"
+        android:minSdkVersion="19"
         android:targetSdkVersion="22" />
```

Comment:
`android:targetSdkVersion` probably should be bumped to match the `28` set elsewhere here too.